scripts/common.py

```diff
@@ -579,6 +581,25 @@ def inplace_replace_file(filename, regex_pattern, replacement_string):
         raise
 
 
+def wait_for_port_to_be_open(port_number):
+    """Wait until the port is open.
```

Comment:
I know you copied this, but it might be nice to clarify a few things for the reader:
- How long does it wait?
- What happens when the time is exceeded?